frontend/src/components/NewRunParametersV2.test.tsx

```diff
@@ -281,6 +281,146 @@ describe('NewRunParametersV2', () => {
     });
   });
 
+  it('test convertInput function for LIST type with default value', () => {
+    const handleParameterChangeSpy = jest.fn();
+    const props = {
+      titleMessage: 'default Title',
+      pipelineRoot: 'defalut pipelineRoot',
+      specParameters: {
+        listParam: {
+          parameterType: ParameterType_ParameterTypeEnum.LIST,
+          defaultValue: [1, 2, 3],
+        },
+      },
+      handlePipelineRootChange: jest.fn(),
+      handleParameterChange: handleParameterChangeSpy,
+    };
+    render(<NewRunParametersV2 {...props}></NewRunParametersV2>);
+
+    const listParam = screen.getByDisplayValue('[1,2,3]');
+    fireEvent.change(listParam, { target: { value: '[4,5,6]' } });
+    expect(handleParameterChangeSpy).toHaveBeenCalledTimes(1);
+    expect(handleParameterChangeSpy).toHaveBeenLastCalledWith({
+      listParam: [4, 5, 6],
+    });
+  });
+
+  it('test convertInput function for LIST type without default value', () => {
+    const handleParameterChangeSpy = jest.fn();
+    const props = {
+      titleMessage: 'default Title',
+      pipelineRoot: 'defalut pipelineRoot',
+      specParameters: {
+        listParam: {
+          parameterType: ParameterType_ParameterTypeEnum.LIST,
+        },
+      },
+      handlePipelineRootChange: jest.fn(),
+      handleParameterChange: handleParameterChangeSpy,
+    };
+    render(<NewRunParametersV2 {...props}></NewRunParametersV2>);
+
+    const listParam = screen.getByLabelText('listParam - LIST');
+    fireEvent.change(listParam, { target: { value: '[4,5,6]' } });
```

Comment:
What is the behavior if user click `Open JSON editor` in this case? We will erase the data in textbox?

Reply:
No, we won't erase the data in the textbox.

Reply:
Sorry I was highlighting a different line. I was thinking about the case when `[4,5,6` is provided, what will happen if user click `Open JSON editor`. Since you mentioned that you are going to handle the error cases in next PR, I think this comment will apply to next PR.